gitversion.yml

```diff
@@ -3,7 +3,7 @@ mode: ContinuousDeployment
 next-version: 1.0.0
 continuous-delivery-fallback-tag: ""
 branches:
-  master:
+  main:
```

Comment:
hmmm you might have to double check this one to see whether that's an actual GitVersion configuration.

Reply:
yes they use main in their documentation